src/main/java/ch/heigvd/statique/Statique.java

```diff
@@ -1,9 +1,9 @@
 package ch.heigvd.statique;
 
-import ch.heigvd.statique.commands.Build;
-import ch.heigvd.statique.commands.Clean;
-import ch.heigvd.statique.commands.Init;
-import ch.heigvd.statique.commands.Serve;
+import ch.heigvd.statique.commands.*;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
```

Comment:
Need to add this import

```suggestion
import java.io.InputStream;
```